src/map/skill.c

```diff
@@ -94,7 +95,8 @@ int skill_splash_target(struct block_list* bl)
 }
 
 /// Returns the id of the skill, or 0 if not found.
-int skill_name2id(const char* name) {
+int skill_name2id(const char* name)
```

Comment:
hmm ... so all the declaration that look like this
```
datatype*  variable
```
should be like this?
```
datatype  *variable
````
the declaration on top are actually invalid? I remember I saw a lot of these, and ya it confuse me, i thought it was intended...
and it should be removed/fix?